clientlib/src/main/proto/yelp/nrtsearch/luceneserver.proto

```diff
@@ -583,7 +584,12 @@ message RefreshResponse {
 }
 
 message CommitRequest {
-    string indexName = 1; //index to commit
+  string indexName = 1; //index to commit
+  string serviceName = 2; // remote storage namespace qualifier for service
+  string resourceName = 3; //remote storage namespace qualifier for resource e.g. indexName
+  bool completeDirectory = 4; // backup complete directory including all current snapshots if true (may backup corrupt segments if backup is created while indexing is happening), otherwise only backup the required segments and segment files
+  bool stream = 5; // if the built tar should be directly streamed to s3, instead of being written to a file first (experimental)
```

Comment:
All of these parameters from CommitRequest seem unused since we are getting the serviceName and resourceName from the config

Reply:
The same is true with the `BackupIndexRequest` in status_quo today.  Also the [code](https://github.com/Yelp/nrtsearch/blob/622e709e7645d6ca22590eab653841b0ce63ce58/src/main/java/com/yelp/nrtsearch/server/luceneserver/BackupIndexRequestHandler.java#L116) that does the Backup in status_quo uses the serviceName and resourceName from the BackupIndexRequest not from the one in config file.

Reply:
Not sure what you mean, these parameters are unused while those from the `BackupIndexRequest` are in use. And IMO not sending these parameters in the commit request is better to not have to change the indexing client.

Reply:
yes, can remove these.